appengine/images/tests/test_guestbook.py

```diff
@@ -66,30 +60,19 @@ def test_img(self):
         greeting.avatar = b'123'
         greeting.put()
 
-        request = webapp2.Request.blank(
-            '/img?img_id=%s' % greeting.key.urlsafe()
-        )
-        response = request.get_response(main.app)
+        response = self.app.get('/img?img_id=%s' % greeting.key.urlsafe())
 
         self.assertEqual(response.status_int, 200)
 
     def test_img_missing(self):
         # Bogus image id, should get error
-        request = webapp2.Request.blank('/img?img_id=123')
-        response = request.get_response(main.app)
-
-        self.assertEqual(response.status_int, 500)
+        self.app.get('/img?img_id=123', status=500)
```

Comment:
Nothing to do with your commit, but we might want to use named constants for HTTP codes.